tests/clients/layer0_test_client.go

```diff
@@ -113,23 +149,20 @@ func (l *Layer0TestClient) ListTasks() []*models.TaskSummary {
 	return tasks
 }
 
-func (l *Layer0TestClient) ScaleService(id string, scale int) *models.Service {
-	service, err := l.Client.ScaleService(id, scale)
+func (l *Layer0TestClient) UpdateEnvironment(environmentID string, req models.UpdateEnvironmentRequest) string {
+	jobID, err := l.Client.UpdateEnvironment(environmentID, req)
```

Comment:
Updates should probably behave like Creates

Reply:
The only similarity that I can see from the caller's perspective is that the function returns a `jobID`. Is there another similarity I'm overlooking?

Reply:
it should probably wait for the job to complete like in create